src/txadmission.h

```diff
@@ -162,6 +162,31 @@ unsigned int TxAlreadyHave(const CInv &inv);
 // threads from adding new tx into the q.
 void CommitTxToMempool();
 
+/**
+ * Check if transaction will be final in the next block to be created.
+ *
+ * Calls IsFinalTx() with current block height and appropriate block time.
+ *
+ * See consensus/consensus.h for flag definitions.
+ */
+bool CheckFinalTx(const CTransactionRef &tx, int flags = -1, const Snapshot *ss = nullptr);
+
+/*
+ * Check if transaction will be BIP 68 final in the next block to be created.
+ *
+ * Simulates calling SequenceLocks() with data from the tip of the current active chain.
+ * Optionally stores in LockPoints the resulting height and time calculated and the hash
+ * of the block needed for calculation or skips the calculation and uses the LockPoints
+ * passed in for evaluation.
+ * The LockPoints should not be considered valid if CheckSequenceLocks returns false.
+ *
+ * See consensus/consensus.h for flag definitions.
+ */
+bool CheckSequenceLocks(const CTransactionRef &tx,
```

Comment:
Maybe it would be good to explain the extra `Snapshot` parameter both for `CheckSequenceLocks` and `CheckFinalTx` in the docs here?